tony-core/src/main/resources/tony-default.xml

```diff
@@ -164,6 +164,12 @@
     <value>0</value>
   </property>
 
+  <property>
+    <description>Resources that will be localized to the container that executes the job. Delimited by comma.</description>
+    <name>tony.worker.resources</name>
+    <value>hdfs://mycluster:9000/user/tony/resource1,hdfs://mycluster:9000/user/tony/resources2</value>
```

Comment:
Should the default be an empty string? If the above is the default, won't this cause a RuntimeException since this cluster/path won't exist usually?

Reply:
will update

Reply:
we shouldn't need to specify the hdfs uri right? Haven't tested it, but since we're calling fs.copyToLocalFile it should assume src is on hdfs and dst is local?

Reply:
i did revert this change. Maybe a later refactoring rb.